python-package/lightgbm/basic.py

```diff
@@ -1397,6 +1397,50 @@ def reset_parameter(self, params):
                 self.handle,
                 c_str(params_str)))
 
+    def get_parameter(self, is_output_raw=False):
```

Comment:
I thank that `get_parameter` in python API is needed anyway, but please update docstring with the aim of uniformity with other docstrings.

```
"""Get parameters of Booster.

Parameters
----------
is_output_raw : bool, optional (default=False)
    Whether to output raw string.

Returns
-------
param : dict or string
    Parameters of Booster.
    If ``is_output_raw=False``, dictionary of parameters with string key and string value;
    otherwise, string of parameters as concat of "key:value".
"""
```

Reply:
I already revert this. Looking for better solution.